spec/features/accounts_spec.rb

```diff
@@ -16,6 +16,7 @@
       allow(Apartment::Tenant).to receive(:switch).with(account.tenant) do |&block|
         block.call
       end
+      allow_any_instance_of(Account).to receive(:find_or_schedule_jobs)
```

Comment:
Thoughts about this? This spec was failing because of the after_save addition. cc @orangewolf